test/Makefile

```diff
@@ -66,8 +76,9 @@ format_unittest: $(GTEST_LIB) format_unittest.o $(SRC_DIR)/format.o
 	$(CXX) $(LDFLAGS) $^ $(LDLIBS) -o $@
 base64_unittest: $(GTEST_LIB) base64_unittest.o $(SRC_DIR)/base64.o
 	$(CXX) $(LDFLAGS) $^ $(LDLIBS) -o $@
-
+configuration_unittest: $(GTEST_LIB) configuration_unittest.o $(SRC_DIR)/configuration.o $(YAML_CPP_LIBS)
```

Comment:
Need to put `$(YAML_CPP_LIBS)` first, otherwise the build of `configuration.o` will fail because the includes won't find `yaml-cpp/yaml.h`.

Reply:
Done, thanks.